src/site/_data/i18n/paths/identity.yml

```diff
@@ -0,0 +1,72 @@
+title:
+  en: Identity
+  es: Identity
```

Comment:
Please don't add untranslated strings here. It should be sufficient to have only "en" string, if others are not translated yet.

Reply:
Is there a process for requesting translations?